### ISSUE TITLE
color mode

### ISSUE THREAD

**Issue by mustafaGFX** [photopea/photopea#2597]
please work on it, it's very very important, I know it's not easy but it deserves

<img width="490" alt="change-color-mode-ai-e1561566172268" src="https://user-images.githubusercontent.com/70863817/98674343-59c7d380-2361-11eb-84e7-0886417057c0.png">

![6782248_preview](https://user-images.githubusercontent.com/70863817/98674670-d0fd6780-2361-11eb-8122-51ab1a1495bf.png)

**Comment by photopea** [comment]
Hi, it has been suggested many times. I am a bit tired to explain, that various color modes do not give you any advantage. But let me explain it again :D

What color modes would you like to add and what exactly should change if you switch to a different color mode?

**Comment by mustafaGFX** [comment]
the most color mode I would like to add is CMYK because its the most important one to print images, actually it's necessary for this purpose, 

try to print an RGB image and look at the troubling result

**Comment by mustafaGFX** [comment]
my opinion in that CMYK is necessary for publications Designs or printed images

**Comment by photopea** [comment]
I think most of people who work in CMYK mode in Adobe Photoshop do it just because they think that they neet to do it, if they plan to print their work afterwards. It is not true.

I recommend you to read my article about color spaces: https://www.photopea.com/learn/color-spaces

**Comment by mustafaGFX** [comment]
wow
I didn't know that, thanks Ivan

**Comment by MichalSZZ** [comment]
@photopea  I made (with my team) one book in my life (color and B&W photos and text, 500 copies), and all my text and black and white photos wasn't made from CMY. For a better result for that kinde of text/images it should to be K (black) only. How to make real black in print in Photopea? How printer would know that text/grayscale images should to be printed only from K colour?

I know - Photopea isn't DTP software, but believe me - I saw the one sample copy of my book without changing B&W image to K only and that looked a lot worse.
